hfc.py

```diff
@@ -1,4 +1,8 @@
-import requests
+try:
+    import requests
+except Exception:
```

Comment:
Please change it into ImportError. Exception catches every exception occurring (including library errors, even if a library is installed). Try adding it to an except statement right before `except Exception`, for example:
```
try:
    import requests
except ImportError:
    print("You're missing requests module. Please install it using pip")
    exit(1)
except Exception:
    print("An exception has occured while importing the requests module.")
    exit(1)
```